From 2f798e87c10a93235a00e95c38e736f9ff7e991b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Mon, 8 Jun 2015 14:29:03 +0200
Subject: [PATCH 04/10] cmake: Add an ordering dep between HTML & man Sphinx
 targets

Add a dependency between HTML & manpage Sphinx targets to prevent two
instances of Sphinx from running in parallel, and therefore solves race
conditions reusing the same doctree directory.

Bug: https://llvm.org/bugs/show_bug.cgi?id=23781
Patch: https://reviews.llvm.org/D23755
---
 docs/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/docs/CMakeLists.txt b/docs/CMakeLists.txt
index f2cd7e9..179a65b 100644
--- a/docs/CMakeLists.txt
+++ b/docs/CMakeLists.txt
@@ -109,6 +109,9 @@ if (LLVM_ENABLE_SPHINX)
 
     if (${SPHINX_OUTPUT_MAN})
       add_sphinx_target(man llvm)
+      if (${SPHINX_OUTPUT_HTML})
+        add_dependencies(docs-llvm-html docs-llvm-man)
+      endif()
     endif()
 
   endif()
-- 
2.9.3

